PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,12 @@
 **CHECK LIST**
-- [ ] specs added
-- [ ] specs passed
-- [ ] code coverage >90%
+- [x] specs added

```

Comment:
what r you doing?
copy and paste this into PR's description, 
DO NOT EDIT ORIGINAL!

Reply:
@igorkasyanchuk 
can somebody with repo permission [change git description](https://docs.gitlab.com/ce/user/project/description_templates.html) template to this by default?

Reply:
@romaslmd I don't know what to change and where on github.
Everything was OK. I believe that this is his first template and he doesn't understand what to do.
@egorodessa ASK TEAMMATES TO HELP YOU with checkboxes